src/content_mapblock.cpp

```diff
@@ -472,10 +488,11 @@ void mapblock_mesh_generate_special(MeshMakeData *data,
 				for(u32 j=0; j<4; j++)
 				{
 					v3s16 neighbordir = cornerdir - halfdirs[j];
-					content_t content = neighbor_contents[neighbordir];
+
+					NeighbourData &neighbour_data = neighbour_datas[NeighbourToIndex(neighbordir)];
+					content_t& content = neighbour_data.content;
```

Comment:
Why is this content_t a reference here?

Are you modifying it somewhere below?